index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sameed - Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+</head>
+
+
+
+<body>
+
+    <header>
+        <nav>
+            <div class="logo-container">
+                <img src="images for portfolio/logo.jpg" alt="" class="logo">
+                <a class="logo-text">Sameed Mohsin</a>
```

Comment:
You are using `font-size: 35px`
![image](https://user-images.githubusercontent.com/61361037/212486208-913ea3b7-3fa5-4b3e-8caa-efef936f982b.png)

According to FIGMA design, `font-size: 20px`

![image](https://user-images.githubusercontent.com/61361037/212486162-42dc5c9a-1596-468a-8eb1-c967c273c435.png)